### PR TITLE
Temp staging branch

### PR DESCRIPTION
**Pull request by Rutvik2901**
Added Loaders everywhere where it's required.
make minor ui changes like card-layout, modal component and home page also.

### CODE REVIEW

File: clone-hackernews/src/Components/ContainerComponent/Styles/index.css [hunk 1]
```diff
@@ -102,4 +102,12 @@ abbr[title] {
...
     text-overflow: ellipsis;
     min-height: 10%;
+}
+
+.ui.grid>.column:not(.row), .ui.grid>.row>.column {
+    padding-right: 0px;
+}
+
+.ui.stackable.grid>.column:not(.row) {
+    padding: 1rem 0px 1rem 1rem !important;
```
Comment (sandeepPirimid) [inline clone-hackernews/src/Components/ContainerComponent/Styles/index.css:112]: can you avoid `!important` ?

Comment (Rutvik2901) [reply clone-hackernews/src/Components/ContainerComponent/Styles/index.css:112]: i am overriding the semantic ui style.

Comment (sandeepPirimid) [reply clone-hackernews/src/Components/ContainerComponent/Styles/index.css:112]: Are they using `!important`?

Comment (Rutvik2901) [reply clone-hackernews/src/Components/ContainerComponent/Styles/index.css:112]: yes

**Pull request merged**